docs/usage/project.swift.mdx

```diff
@@ -799,6 +801,23 @@ A `Configuration` object describes the build settings and the `.xcconfig` file o
   ]}
 />
 
+### SettingValue
+
+A `SettingValue` can be one of the following options:
+
+<EnumTable
+  cases={[
+    {
+      case: '.string(String)',
+      description: 'A single setting value for example "DEBUG=1".',
+    },
+    {
+      case: '.array([String])',
+      description: 'Multi-value setting value for example ["${inherited}", "DEBUG=1"].',
+    },
+  ]}
+/>
+
```

Comment:
I'd mention that `SettingValue` conforms the `ExpressiveByLiteral..`. protocols and therefore it can be initialized with a string or an array directly.

Reply:
👍 good idea, added an extra line about it.